src/busesApp/updateValidation.js

```diff
@@ -0,0 +1,19 @@
+import { body } from 'express-validator';
+
+const updateValidation = () => [
+	body('busType', 'busType_required')
+		.optional()
+		.notEmpty()
+		.bail()
+		.isLength({ min: 3 })
+		.withMessage('min_3_char'),
+	body('seats')
+		.optional()
+		.notEmpty()
+		.withMessage('seats_required')
+		.bail()
+		.isLength({ min: 2 })
+		.withMessage('min_2_char'),
```

Comment:
will it be possible to update the bus type or seats? (considering the physical aspects of it)

Reply:
Yes, in case there might have been a typo during registration. However, only the operator and admin have the rights to do so.

Reply:
Yes, in case there might have been a typo during the bus registration. However, only the operator and admin will have the rights to do so.